src/modules/discovery.json

```diff
@@ -448,6 +462,10 @@
       "tags": [
         {
           "name": "polymorphic-reducer"
+        },
+        {
+          "name": "capabilities",
+          "x-provides": "xrn:firebolt:capability:discovery:watched"
```

Comment:
Should this be provides?

Reply:
I think capabilities/roles should be based on the intention of the method, not the semantics. This method allows an app to provide us with watch history data, so I think it's a provide role.

Reply:
okay, after a two hour convo in slack you convinced me :)

fixed.